homeassistant/components/lyric/sensor.py

```diff
@@ -0,0 +1,254 @@
+"""Support for Honeywell Lyric sensor platform."""
+from datetime import datetime, timedelta
+
+from aiolyric.objects.device import LyricDevice
+from aiolyric.objects.location import LyricLocation
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_TEMPERATURE,
+    DEVICE_CLASS_TIMESTAMP,
+)
+from homeassistant.helpers.typing import HomeAssistantType
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+from homeassistant.util import dt as dt_util
+
+from . import LyricDeviceEntity
+from .const import (
+    DOMAIN,
+    PRESET_HOLD_UNTIL,
+    PRESET_NO_HOLD,
+    PRESET_PERMANENT_HOLD,
+    PRESET_TEMPORARY_HOLD,
+    PRESET_VACATION_HOLD,
+)
+
+SCAN_INTERVAL = timedelta(seconds=120)
+PARALLEL_UPDATES = 6
```

Comment:
Is this still needed?

Reply:
Probably not if the coordinator handles all updates anyway?

Reply:
```suggestion
```